From bd6746591d0d325933ef99cf67eff755974136d7 Mon Sep 17 00:00:00 2001
Message-Id: <bd6746591d0d325933ef99cf67eff755974136d7.1387276076.git.minovotn@redhat.com>
In-Reply-To: <e610718166120379517e80d1a7aa12d60294209b.1387276076.git.minovotn@redhat.com>
References: <e610718166120379517e80d1a7aa12d60294209b.1387276076.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 10 Dec 2013 15:12:51 +0100
Subject: [PATCH 10/16] slavio_misc: Fix slavio_led_mem_readw/_writew
 tracepoints

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1386688376-29521-3-git-send-email-armbru@redhat.com>
Patchwork-id: 56113
O-Subject: [PATCH 7.0 qemu-kvm 2/7] slavio_misc: Fix slavio_led_mem_readw/_writew tracepoints
Bugzilla: 997832
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

From: Markus Armbruster <armbru@redhat.com>

Broken since they got added in commit 97bf485.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
(cherry picked from commit f3a64b8c89ac9b5111f97e2653d249e4668764b3)
---
 hw/misc/slavio_misc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/misc/slavio_misc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/misc/slavio_misc.c b/hw/misc/slavio_misc.c
index a7a9368..3571850 100644
--- a/hw/misc/slavio_misc.c
+++ b/hw/misc/slavio_misc.c
@@ -368,7 +368,7 @@ static void slavio_led_mem_writew(void *opaque, hwaddr addr,
 {
     MiscState *s = opaque;
 
-    trace_slavio_led_mem_readw(val & 0xffff);
+    trace_slavio_led_mem_writew(val & 0xffff);
     switch (addr) {
     case 0:
         s->leds = val;
-- 
1.7.11.7

